src/opencast.js

```diff
@@ -259,10 +301,11 @@ export class Opencast {
 
   // Adds the DC Catalog with the given metadata to the current ingest process
   // via `ingest/addDCCatalog`. Do not call this method outside of `upload`!
-  async addDcCatalog({ mediaPackage, title, creator, uploadSettings }) {
-    const seriesId = uploadSettings?.seriesId;
+  async addDcCatalog({ mediaPackage, title, creator, description, seriesID, uploadSettings }) {
+    //const seriesId = uploadSettings?.seriesId;
+    let username = this.#currentUser.user.username;
 
-    const dcc = dcCatalog({ creator, title, seriesId });
+    const dcc = dcCatalog({ creator, title,description, seriesID, username });
```

Comment:
You currently completely ignore the `seriesId` set in the settings. The settings series ID should override any user provided one. The series selection shouldn't be shown if the series ID is already provided via settings.